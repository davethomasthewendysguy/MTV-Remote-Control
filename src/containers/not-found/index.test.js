// eslint-disable-next-line no-unused-vars
import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from '.';

describe('NotFound component', () => {
	it('should render successfully', () => {
		const tree = renderer.create(<NotFound />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
