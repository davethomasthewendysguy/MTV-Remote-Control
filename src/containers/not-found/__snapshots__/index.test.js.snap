// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render successfully 1`] = `
<h1
  className="not-found__Title-gPmEgc cFWGSp"
>
  404 
  <br />
  Page not found
</h1>
`;
